deepchem/data/datasets.py

```diff
@@ -308,21 +462,33 @@ def gen_data():
   def make_pytorch_dataset(self, epochs=1, deterministic=False):
     """Create a torch.utils.data.IterableDataset that iterates over the data in this Dataset.
 
-    Each value returned by the Dataset's iterator is a tuple of (X, y, w, id) for
-    one sample.
+    Each value returned by the Dataset's iterator is a tuple of (X, y,
+    w, id) for one sample.
 
     Parameters
     ----------
     epochs: int
       the number of times to iterate over the Dataset
     deterministic: bool
-      if True, the data is produced in order.  If False, a different random
-      permutation of the data is used for each epoch.
+      if True, the data is produced in order.  If False, a different
+      random permutation of the data is used for each epoch.
+
+    Returns
+    -------
+    `torch.utils.data.IterableDataset` that iterates over the data in
+    this dataset.
     """
     raise NotImplementedError()
 
   def to_dataframe(self):
-    """Construct a pandas DataFrame containing the data from this Dataset."""
+    """Construct a pandas DataFrame containing the data from this Dataset.
+
+    Returns
+    -------
+    pandas datafarme. Will have column "X1,X2,..." for features,
+    "y1,y2,..." for labels, "w1,w2,..." for weights, and column "ids"
+    for identifiers.
```

Comment:
Not necessarily.  If `X`, `y`, or `w` only has a single column, the dataframe will just have a column called "X", "y", or "w".

Reply:
Good catch. Added in a more exhaustive comment here.